### PR TITLE
Typos fixed and copypasta gone

### PR DESCRIPTION
**Pull request by QuantumAlmonds**
Fixed some typos and replaced a long if/else statement with a switch statement.

### CODE REVIEW

File: libhw/adc.c [hunk 1]
```diff
@@ -26,58 +26,123 @@
...
+  //   send[0] = 0x40;
+  //   send[1] = 0x1E;
+  // } else if (channel == 13) {
+  //   send[0] = 0xC0;
+  //   send[1] = 0x1E;
+  // } else if (channel == 14) {
+  //   send[0] = 0x40;
+  //   send[1] = 0x1F;
+  // } else {
+  //   send[0] = 0xC0;
```
Comment (EdHurtig) [inline libhw/adc.c]: Lots of comment, pls purge

**Review by EdHurtig — APPROVED** [review]
Approved pending comments

File: libhw/adc.c (hunk 1 above)
Comment (QuantumAlmonds) [reply libhw/adc.c]: Will do

**Comment by QuantumAlmonds** [comment]
Fixes #70

**Pull request merged**